keel/api/v1/auth/serializers.py

```diff
@@ -161,6 +162,15 @@ def get_doc_type(self, obj):
     def get_orignal_file_name(self, obj):
         return obj.doc.original_name
 
+    def get_user_type(self, obj):
+        user_id = obj.doc.owner_id
+        try:
+            user = User.objects.get(id = user_id)
+        except Exception as e:
+            log_error("ERROR", "ListUserDocumentSerializer: get_user_type",str(user_id), err = str(e))
+            return dict(User.USER_TYPE_CHOICES).get(User.CUSTOMER)
```

Comment:
ok...so are we sending customer type user by default!!

Reply:
I did find it justified to raise errors in such cases. so sending default of Customer. We can send emptystring, that's for FE to handle then. 

Alas, we can raise error